core/templates/dev/head/pages/topics_and_skills_dashboard/topics_and_skills_dashboard.html

```diff
@@ -16,123 +16,42 @@
-          <a class="oppia-dashboard-tabs-text"
+            ng-class="{'dashboard-tabs-active': activeTab === TAB_NAME_TOPICS}">
+          <a class="dashboard-tabs-text"
              ng-click="setActiveTab(TAB_NAME_TOPICS)"
              translate="I18N_DASHBOARD_TOPICS">
           </a>
         </li>
         <li ng-if="skillSummaries.length > 0"
-            ng-class="{'oppia-dashboard-tabs-active': activeTab === TAB_NAME_SKILLS}">
-          <a class="oppia-dashboard-tabs-text"
+            ng-class="{'dashboard-tabs-active': activeTab === TAB_NAME_SKILLS}">
+          <a class="dashboard-tabs-text"
              ng-click="setActiveTab(TAB_NAME_SKILLS)"
              translate="I18N_DASHBOARD_SKILLS">
           </a>
         </li>
       </ul>
-    </div>
 
-    <div class="oppia-dashboard-container" ng-if="activeTab === TAB_NAME_TOPICS">
-      <md-card layout="row"
-               class="oppia-dashboard-list-view-item">
-        <table class="oppia-dashboard-table">
-            <colgroup>
-              <col style="width: 28%;">
-              <col style="width: 24%;">
-              <col style="width: 24%;">
-              <col style="width: 24%;">
-            </colgroup>
-          <tr>
-            <th ng-repeat="key in TOPIC_HEADINGS"
-                class="oppia-dashboard-table-headings">
-              <p ng-if="key === 'name'" translate="I18N_DASHBOARD_TABLE_HEADING_TOPIC_NAME"></p>
-              <p ng-if="key === 'subtopic_count'" translate="I18N_DASHBOARD_TABLE_HEADING_SUBTOPICS"></p>
-              <p ng-if="key === 'skill_count'" translate="I18N_DASHBOARD_TABLE_HEADING_SKILLS"></p>
-              <p ng-if="key === 'canonical_story_count'" translate="I18N_DASHBOARD_TABLE_HEADING_CANONICAL_STORIES"></p>
-            </th>
-          </tr>
-          <tr ng-repeat="topic in topicSummaries"
-              class="list-item">
-            <td>
-              <a ng-href="<[getTopicEditorUrl(topic.id)]>" class="oppia-dashboard-list-summary">
-                <span> <[topic.name]> </span>
-              </a>
-            </td>
-            <td>
-              <a ng-href="<[getTopicEditorUrl(topic.id)]>" class="oppia-dashboard-list-summary">
-                <span> <[topic.subtopic_count]> </span>
-              </a>
-            </td>
-            <td>
-              <a ng-href="<[getTopicEditorUrl(topic.id)]>" class="oppia-dashboard-list-summary">
-                <span> <[topic.total_skill_count]> </span>
-                <span ng-if="topic.uncategorized_skill_count > 0"> ( <[topic.uncategorized_skill_count]> uncategorized ) </span>
-              </a>
-            </td>
-            <td>
-              <a ng-href="<[getTopicEditorUrl(topic.id)]>" class="oppia-dashboard-list-summary">
-                <span> <[topic.canonical_story_count]> </span>
-              </a>
-            </td>
-          </tr>
-        </table>
-      </md-card>
+    <div ng-if="activeTab === TAB_NAME_TOPICS">
```

Comment:
Something's up with the indentation here, it does't look like a sibling of the previous line (though it is).

/cc @apb7 @kevinlee12 we should catch this automatically.

Reply:
Done.